frontend/app/src/components/profitloss/ProfitLossEvents.vue

```diff
@@ -180,22 +182,27 @@ const checkGroupLine = (entries: ProfitLossEvents, index: number) => {
       sort-by="time"
     >
       <template #item.group="{ item }">
-        <VTooltip v-if="item.groupId" right>
-          <template #activator="{ on, attrs }">
-            <div v-bind="attrs" :class="$style['group']" v-on="on">
+        <RuiTooltip
+          v-if="item.groupId && (item.groupLine.top || item.groupLine.bottom)"
+          :popper="{ placement: 'right' }"
+          open-delay="400"
+          class="h-full"
+        >
+          <template #activator>
+            <div :class="css.group">
               <div
                 v-if="item.groupLine.top"
-                :class="`${$style['group__line']} ${$style['group__line-top']}`"
+                :class="`${css.group__line} ${css['group__line-top']}`"
               />
-              <div :class="$style['group__dot']" />
+              <div :class="css.group__dot" />
               <div
                 v-if="item.groupLine.bottom"
-                :class="`${$style['group__line']} ${$style['group__line-bottom']}`"
+                :class="`${css.group__line} ${css['group__line-bottom']}`"
```

Comment:
since you're not adding any string, can use the array notation:
```html
:class="[css.group__line, css['group__line-top']]"
:class="[css.group__line, css['group__line-bottom']]"
```